docs/fsharp/tutorials/type-providers/accessing-a-sql-database-entities.md

```diff
@@ -183,17 +183,17 @@ let fullContext = context.DataContext
 let nullable value = new System.Nullable<_>(value)
 
 let addInstructor(lastName, firstName, hireDate, office) =
-let hireDate = DateTime.Parse(hireDate)
-let newPerson = new EntityConnection.ServiceTypes.Person(LastName = lastName,
+  let hireDate = DateTime.Parse(hireDate)
+  let newPerson = new EntityConnection.ServiceTypes.Person(LastName = lastName,
                                                          FirstName = firstName,
                                                          HireDate = nullable hireDate)
```

Comment:
Now you just wait until someone from Microsoft (read: not me) comes along and either accepts the PR or tells you what else to improve in the PR (but with such a small PR, I think there's not much room for that).